db/seeds.rb

```diff
@@ -51,32 +51,55 @@ def random_record_for_org(org, klass)
 # ----------------------------------------------------------------------------
 
 pdx_org = Organization.find_or_create_by!(short_name: "diaper_bank") do |organization|
-  organization.name    = "Pawnee Diaper Bank"
-  organization.street  = "P.O. Box 22613"
-  organization.city    = "Pawnee"
-  organization.state   = "Indiana"
+  organization.name = "Pawnee Diaper Bank"
+  organization.street = "P.O. Box 22613"
+  organization.city = "Pawnee"
+  organization.state = "Indiana"
   organization.zipcode = "12345"
-  organization.email   = "info@pawneediaper.org"
+  organization.email = "info@pawneediaper.org"
 end
 Organization.seed_items(pdx_org)
 
 sf_org = Organization.find_or_create_by!(short_name: "sf_bank") do |organization|
-  organization.name    = "SF Diaper Bank"
-  organization.street  = "P.O. Box 12345"
-  organization.city    = "San Francisco"
-  organization.state   = "CA"
+  organization.name = "SF Diaper Bank"
+  organization.street = "P.O. Box 12345"
+  organization.city = "San Francisco"
+  organization.state = "CA"
   organization.zipcode = "90210"
-  organization.email   = "info@sfdiaperbank.org"
+  organization.email = "info@sfdiaperbank.org"
 end
 Organization.seed_items(sf_org)
 
+sc_org = Organization.find_or_create_by!(short_name: "sc_bank") do |organization|
+  organization.name = "Second City Essentials Bank"
+  organization.street = Faker::Address.street_address
+  organization.city = Faker::Address.city
+  organization.state = Faker::Address.state_abbr
+  organization.zipcode = Faker::Address.zip_code
+  organization.email = "info@scdiaperbank.org"
+end
+Organization.seed_items(sc_org)
+
+# The list of organizations that will have donations, purchases, requests, distributions,
+# and the records those rely on generated.
+complete_orgs = [pdx_org, sc_org]
+
 # At least one of the items is marked as inactive
-Organization.all.each do |org|
+Organization.all.find_each do |org|
```

Comment:
Using `find_each` instead of `each` was actually a change made by the linter. If the use of `each` was intentional, perhaps we need to configure the linter to not automatically make this change?

Reply:
`find_each` is pretty much always a better approach than `each`. There isn't really a case where the other way around is better.